frontend/public/components/monitoring/receiver-forms/alert-manager-receiver-forms.tsx

```diff
@@ -580,3 +600,7 @@ type SaveAsDefaultCheckboxProps = {
   dispatchFormChange: Function;
   tooltip: string;
 };
+
+type ReceiverInfoTipProps = {
+  type: InitialReceivers.Watchdog | InitialReceivers.Critical | InitialReceivers.Default;
```

Comment:
Shouldn't this just be:

```suggestion
  type: InitialReceivers;
```

Reply:
yup, thanks, fixed :-)